rs/ic_os/config/src/generate_testnet_config.rs

```diff
@@ -171,9 +171,13 @@ fn create_guestos_config(config: GenerateTestnetConfigArgs) -> Result<GuestOSCon
 
     let deployment_environment = deployment_environment.unwrap_or(DeploymentEnvironment::Testnet);
 
-    let logging = Logging {
-        elasticsearch_hosts: elasticsearch_hosts.unwrap_or_else(|| "".to_string()),
-        elasticsearch_tags,
+    let logging = if elasticsearch_hosts.is_some() {
```

Comment:
nit: I'm not sure that it is cleaner, but I've got Rust on the mind 😅.

```rust
let logging = elasticsearch_hosts.map(|elasticsearch_hosts| Logging { elasticsearch_hosts, elasticsearch_tags }).unwrap_or_defautl();
```

Reply:
I was being silly before. It can just be:
```
let logging = Logging {
        elasticsearch_hosts,
        elasticsearch_tags,
    };
```
😄  8cae4c333b191e19b20633b4c961c45e6ee99515